ext/snmp/php_snmp.h

```diff
@@ -92,8 +92,7 @@ ZEND_END_MODULE_GLOBALS(snmp)
 #define SNMP_G(v) (snmp_globals.v)
 #endif
 
-#define REGISTER_SNMP_CLASS_CONST_LONG(const_name, value) \
-	zend_declare_class_constant_long(php_snmp_ce, const_name, sizeof(const_name)-1, (zend_long)value);
+#define LONG_CONST(c) (zend_long) c;
```

Comment:
These are supposed to be *class* constants, e.g. `SNMP::VERSION_1`.

Reply:
Yep, some of them became class constants (https://github.com/php/php-src/pull/9113/files#diff-1b95a9a4e58dd9c0039fe9f2044e8c9d23ef92cd1d3cfa66e5d953f555c987b7R190). I only added this macro because all the constants were originally casted to `zend_long`.

Reply:
I probably realized what you meant: since `REGISTER_SNMP_CLASS_CONST_LONG` is only used for class constants, `LONG_CONST()` is not needed for global constants. However, based on a quick look at the class constants values, it seems to me that neither the cast is needed, so I've just dropped them.